### PR TITLE
Add Ubuntu Dockerfile and fix entrypoint to work in Ubuntu

### PR DESCRIPTION
**Pull request by redhatrises**

### CODE REVIEW

File: Dockerfile.ubuntu [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    libnl-3-200 net-tools zip openssl hostname iproute2 /tmp/falcon-sensor.deb && \
+    rm -rf /tmp/falcon-sensor.deb /var/lib/apt/lists/* && \
+    apt clean
+
+#
+# Copy the entrypoint script into the container and make sure
+# that its executable. Add the symlink for backwards compatability
+#
+COPY entrypoint.sh /usr/local/bin/
+RUN ln -s /usr/local/bin/entrypoint.sh /
```
Comment (shawndwells) [inline Dockerfile.ubuntu:52]: With ``entrypoint`` updated to ``/usr/local/bin/entrypoint.sh``, is this needed anymore?

Comment (redhatrises) [reply Dockerfile.ubuntu:52]: Probably not... should we even install `entrypoint.sh` in `/usr/local/bin` vs `/`?

Comment (shawndwells) [reply Dockerfile.ubuntu:52]: your call, Mr. Maintainer! :)

Comment (redhatrises) [reply Dockerfile.ubuntu]: hahahaha.... okay! It's updated.

**Review by shawndwells — APPROVED** [review]

**Pull request merged**